### PR TITLE
[CNVS Upgrade] Audit table styles

### PR DESCRIPTION
**Pull request by jfurrow**
@ashenden I left two `TODO` notes in here, let me know what you think.

My overall goals with the modifications to `table.less` are to remove any styles that don't specifically pertain to the table, reduce specificity, and clearly identify where classes do relate to the table (e.g. the `heading` vs `headline` classes which actually indicate sorted and emphasized table cells are now called `table-cell-sorted` and `table-cell-emphasized`).

### CODE REVIEW

**Comment by ashenden** [comment]
Reviewing the code changes now.  Generally, I'd love to see the `tables` object make it's way under `/content` instead of `/component` to mirror how it's organized in CNVS (https://cl.ly/2N3w310R1m3y).

File: src/js/components/PackagesTable.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ class PackagesTable extends React.Component {
 
   getClassName(prop, sortBy, row) {
     return classNames({
-      'highlight': prop === sortBy.prop,
+      'table-cell-sorted': prop === sortBy.prop,
```
Comment (ashenden) [inline src/js/components/PackagesTable.js]: The `table-cell-sorted` should probably just be replaced with `active`.  See http://localhost:4200/#/styles/content/tables/ for reference.

Comment (jfurrow) [inline src/js/components/PackagesTable.js]: Done!

**Comment by jfurrow** [comment]
@ashenden I moved the table styles from `components/tables` to `content/tables`

**Comment by ashenden** [comment]
Looks good to me.  Merge that shit!

**Pull request merged**